docs/upgrade/v1-2-1-to-v1-2-2.md

```diff
@@ -0,0 +1,15 @@
+---
+sidebar_position: 2
+sidebar_label: Upgrade from v1.2.1 to v1.2.2
+title: "Upgrade from v1.2.1 to v1.2.2"
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.3/upgrade/v1-2-1-to-v1-2-2"/>
+</head>
+
+## General information
+
+Once there is an upgradable version, the Harvester GUI Dashboard page will show an upgrade button. For more details, please refer to [start an upgrade](./automatic.md#start-an-upgrade).
+
+For the air-gap env upgrade, please refer to [prepare an air-gapped upgrade](./automatic.md#prepare-an-air-gapped-upgrade).
```

Comment:
```suggestion
An **Upgrade** button appears on the **Dashboard** screen whenever a new Harvester version that you can upgrade to becomes available. For more information, see [Start an upgrade](./automatic.md#start-an-upgrade).

For air-gapped environments, see [Prepare an air-gapped upgrade](./automatic.md#prepare-an-air-gapped-upgrade).
```

Reply:
This wording is from [Upgrade from v1.1.1/v1.1.2 to v1.1.3](https://docs.harvesterhci.io/v1.4/upgrade/v1-1-1-to-v1-1-3#general-information).

Reply:
Fixed. Thanks for adding comments to all versions!